tests/Aql/MultiDependencySingleRowFetcherTest.cpp

```diff
@@ -710,7 +714,7 @@ TEST_P(MultiDependencySingleRowFetcherTest,
   // This has to be done by the InputRange hold externally.
   // It has seperate tests
   std::vector<std::deque<SharedAqlItemBlockPtr>> data;
-  for (size_t i = 0; i < numberDependencies(); ++i) {
+  for (int i = 0; i < numberDependencies(); ++i) {
```

Comment:
is this intended? is `numberDependencies()` an `int` that needs a sign?

Reply:
no, but `i` needs to be signed ... this is just a quick and dirty fix. please fix